dcc-import-gene/src/main/java/org/icgc/dcc/imports/gene/GeneImporter.java

```diff
@@ -43,12 +49,12 @@
 public class GeneImporter implements SourceImporter {
 
   @NonNull
-  private final URI genesBsonUri;
+  private final URI gtfUri;
   @NonNull
   private final MongoClientURI mongoUri;
 
-  public GeneImporter(@NonNull MongoClientURI mongoUri, @NonNull URI genesBsonUri) {
-    this.genesBsonUri = genesBsonUri;
+  public GeneImporter(@NonNull MongoClientURI mongoUri, @NonNull URI gtfUri) {
```

Comment:
This constructor shouldn't be needed due to `@RequiredArgsConstructor`. Please remove